concourse/scripts/self-update-pipeline.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - AWS_ACCOUNT
+# - SELF_UPDATE_PIPELINE
+#
+# Optional variables:
+# - BRANCH
+
+set -u
+set -e
+
+if [ ! -d "./paas-bootstrap" ]; then
+  echo "Resource paas-bootstrap must be checked out"
+  exit 1
+fi
```

Comment:
Removed. Looks like it was originally copied from the [paas-cf implementation](https://github.com/alphagov/paas-cf/blob/4200fb86f90666531956b6043690094e283f530a/concourse/scripts/self-update-pipeline.sh#L15). I'll raise it there too.